src/ClientConfig_TEST.cc

```diff
@@ -35,11 +35,7 @@ using namespace fuel_tools;
 /// \param[in] _path Path to file to be removed.
 void removeFileTemp(const std::string &_path)
 {
-#ifndef _WIN32
-  EXPECT_TRUE(ignition::common::removeFile(_path));
-#else
   ignition::common::removeFile(_path);
-#endif
```

Comment:
Is this not returning true on Linux and macOS now? Ideally we'd update this so it checks it's true for all platforms.